changelog.md

```diff
@@ -5,18 +5,29 @@
 * e2e Tests HTML Report
 * Add new service with specified plugin
 * `./make_deb` script to create debian package (in `packages/Debian/build)
+* In auth mandatory mode FogLAMP running status on top will show the lock icon button until not logged into system
```

Comment:
In auth mandatory mode, if allowPing is false, FogLAMP running status on top will show the lock icon button until not logged into system